examples/scene/point_cloud.py

```diff
@@ -23,20 +23,21 @@
 
 
 # generate data
-pos = np.random.normal(size=(100000, 3), scale=0.2)
+pos = np.random.normal(size=(10000, 3), scale=0.2)
```

Comment:
Are the changes in this example needed or was this accidentally committed?

Reply:
Oops, reacted to the wrong thing by mistake. This particular change was accidental (no performance hit that big, no worries! :)). There are a couple of changes that were on purpose, specifically on the scaling, due to some inconsistencies. I still need to debug exactly what is going on.

Reply:
Ok, everything is fixed now, no changes needed.